### ISSUE TITLE
Support for other indoor tags

### ISSUE THREAD

**Issue by saerdnaer** [yarl/osmtools-indoor#13]
Hi, is there a reason why the tags room=\* or area=\* are not supported?

**Comment by yarl** [comment]
Hi, I don't want to mix standards. Can you show me a building with these tags implemented?

**Comment by saerdnaer** [comment]
Of course: http://www.openstreetmap.org/?lat=48.262437&lon=11.667689&zoom=18

For more places with indoor coverage see http://wiki.osm.org/Indoor#Places
